net-utils/src/ip_echo_server.rs

```diff
@@ -22,8 +22,7 @@ pub type IpEchoServer = Runtime;
 // Enforce a minimum of two threads:
 // - One thread to monitor the TcpListener and spawn async tasks
 // - One thread to service the spawned tasks
-// The unsafe is safe because we're using a fixed, known non-zero value
-pub const MINIMUM_IP_ECHO_SERVER_THREADS: NonZeroUsize = unsafe { NonZeroUsize::new_unchecked(2) };
+pub const MINIMUM_IP_ECHO_SERVER_THREADS: NonZeroUsize = NonZeroUsize::new(2).unwrap();
```

Comment:
This does not compile.
Also cc @yihau because surprisingly the CI did not complain.

```
error: `std::option::Option::<T>::unwrap` is not yet stable as a const fn
  --> net-utils/src/ip_echo_server.rs:25:58
   |
25 | pub const MINIMUM_IP_ECHO_SERVER_THREADS: NonZeroUsize = NonZeroUsize::new(2).unwrap();
   |                                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: add `#![feature(const_option)]` to the crate attributes to enable
```

Reply:
Wait... on which compiler version does it not compile?

Reply:
We just recently bumped up rust version.
So this is actually good.
Please ignore.